src/modules/universe/RadixNodeDiscoveryFromNodeFinder.ts

```diff
@@ -15,31 +16,36 @@ export default class RadixNodeDiscoveryFromNodeFinder implements RadixNodeDiscov
      */
     constructor(
         readonly bootstrapService: string, 
-        readonly nodeRPCAddress: (nodeIp: string) => string) {}
+        readonly wsAddress: (host: string, port: number) => string,
+        readonly httpAddress: (host: string, port: number) => string) {}
 
     public async loadNodes() {
         const bootstrapNodeIP = (await axios.get(this.bootstrapService)).data
         if (bootstrapNodeIP.length < 2) {
             throw new Error('Failed to get node list from bootstrap service')
         }
-        const bootstrapNode = this.nodeRPCAddress(bootstrapNodeIP)
+        const bootstrapNode = this.httpAddress(bootstrapNodeIP, 0)
 
         const getPeersRequestData = {
             id: 0,
             method: 'Network.getLivePeers',
-            params: []
+            params: [],
         }
         const nodeListResponse = await axios.post(
             bootstrapNode,
-            getPeersRequestData
+            getPeersRequestData,
```

Comment:
I've noticed you add a trailing ','-sign on the last field as well. Is this convenience or is there some magic behind it too?

Reply:
I have tslint set up so that it complains if you don't. The magic happens in git merges when you need to add an extra parameter/list item/object property after the last one, especially if you need to add 2 different items in 2 different branches.